tests/playwright/shiny/inputs/input_task_button/test_input_task_button.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+from conftest import ShinyAppProc
+from controls import InputNumeric, InputTaskButton, OutputText
+from playwright.sync_api import Page
+
+
+def click_extended_task_button(
+    button: InputTaskButton,
+    current_time: OutputText,
+    button_label: list[str],
+) -> str:
+    button.expect_state("ready")
+    button.expect_label_text(button_label)
```

Comment:
Let's move this out of the function and have it live in the test code directly. (It is unrelated to clicking the button)